.gitlab-ci.yml

```diff
@@ -100,7 +100,7 @@ test-android-rust-stable:
 test-windows-rust-stable:
   stage:                           test
   cache:
-    key:                           "%CI_JOB_NAME%"
+    key:                           "${CI_JOB_NAME}"
     paths:
       - "%CI_PROJECT_DIR%/target/"
       - "%CI_PROJECT_DIR%/cargo/"
```

Comment:
Do we need to change the notation everywhere from `%VAR%` to `${VAR}` ?

Reply:
As far as I understand — no, since scripts for caching are evaluated in linux environment (on a Gitlab server side) when we convert yaml into an actual executable; but _paths_ are expanded on the target machine when it's actually executed.

But I might as well be mistaken